fastn-update/src/lib.rs

```diff
@@ -249,6 +249,8 @@ async fn get_manifest(
     .context(DownloadManifestSnafu {
         package: package_name,
     })?;
+    // Append newline character
+    manifest_bytes.push(b'\n');
```

Comment:
No.

Reply:
DO NOT accept garbage. Do not fix other peoples garbage. Call things out. This is defensive programming, "oh I know they were supposed to generate the manifest file with new line in the end, but they screwed up, let me fix it for them". It you on both sides. You are writing code to generate manifest.json and you are writing code for downloading it. If you are generating manifest.json, do you have to add new line after downloading it?